domain/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
```

Comment:
Domain module should be independent from the platform (Android in this case) it has to be only a kotlin module without any knowledge about Android dependency

Reply:
I changed it and now I created the module `Java or Kotlin Library` type